features/order/managing_orders/seeing_order_discounts.feature

```diff
@@ -0,0 +1,47 @@
+@managing_orders
+Feature: Seeing discounts of an order
+    In order to see discount on specific order
```

Comment:
In order to be aware of the amount of discount applied to an order
...
I want to see the discount value of a specific order